packages/react-query-4/src/SuspenseInfiniteQuery.tsx

```diff
@@ -6,6 +6,35 @@ import {
   useSuspenseInfiniteQuery,
 } from './useSuspenseInfiniteQuery'
 
+/**
+ * We provide these components to clearly express what causes suspense at the same depth.
+ * `<SuspenseInfiniteQuery/>` serves to make `useSuspenseInfiniteQuery` easier to use in jsx.
+ *
+ * @see {@link https://suspensive.org/en/docs/react-query/SuspenseInfiniteQuery Suspensive Docs}
+ *
+ * @example
+ * ```tsx
+ * import { SuspenseInfiniteQuery } from '@suspensive/react-query'
+ *
+ * <SuspenseInfiniteQuery {...postsInfiniteQueryOptions(userId)}>
+ *   {({ data, fetchNextPage }) => (
+ *     <>
+ *       {data.pages.map((post) => (
+ *         <PostListItem {...post} />
+ *       ))}
+ *       <button
+ *         type="button"
+ *         onClick={() => {
+ *           fetchNextPage()
+ *         }}
+ *       >
+ *         Load More
+ *       </button>
+ *     </>
+ *   )}
+ * </SuspenseInfiniteQuery>
+ * ```
```

Comment:
Could we remove examples in jsdoc? I want to make our https://suspensive.org as ssot to see good examples

So I think link is good! I agree offline library user will get not good DX. but in my opinion, reducing our maintaining fee is also important and I want to focus other thing more.

Reply:
> If you can care it consistently with me, I want to rely on you! Let's do it and Take care it continuously

Of course!

I will ensure that the examples for the component API are at a level that users can understand, providing just enough to grasp the concepts, so that there are minimal maintenance costs.